active/0000-macro-blueprint.md

```diff
@@ -0,0 +1,47 @@
+# macro blueprint RFC
+
+- Start Date: 2016-09-12
+- RFC PR: (leave this empty)
+- ember-cli Issue: (leave this empty)
+
+# Summary
+
+Add a `macro` blueprint to the default list of blueprints.
+
+# Motivation
+
+To make it slightly easier for people to make custom computed macros. The syntax of a custom macro is already dead simple, but having a default structure, starting point, and folder location would help beginners get started.
+
+# Detailed design
+
+The new blueprint would be added to `ember-cli-legacy-blueprints` since the ember addon is not ready yet. It would generate a file that looks like this:
+
+``` js
+import Ember from 'ember';
+
+export default function(/* keys... */) {
+  return Ember.computed(/* keys... */, function() {
+    // your normal computed code
+  });
+}
+```
+Then once filled in, could be used like:
+
+``` js
+import myMacro from 'my-project/macros/my-macro';
```

Comment:
This works in terms of module unification, I just dislike squatting on another top-level collection.